### PR TITLE
Make --generate-debug-info hidden

### PR DESCRIPTION
**Pull request by seanyoung**
This feature is incomplete and does not have any users yet. For example, there is a debugging endpoint for Solana in the works, but this is not finished yet. Also, this will be useful for static analysis, which requires debug information for higher-level type information.

### CODE REVIEW

File: src/bin/solang.rs [hunk 1]
```diff
@@ -878,19 +858,45 @@ fn imports_arg(matches: &ArgMatches) -> FileResolver {
...
+fn parse_import_map(map: &str) -> Result<(String, String), String> {
+    if let Some((var, value)) = map.split_once('=') {
+        Ok((var.to_owned(), value.to_owned()))
+    } else {
+        Err("contains no '='".to_owned())
+    }
+}
+
+#[derive(Serialize)]
+pub struct EwasmContract {
```
Comment (LucasSte) [inline src/bin/solang.rs:882]: Why did you move these structs down?

Comment (seanyoung) [reply src/bin/solang.rs:882]: These structs are the serialization format for `--standard-json`. I don't think they belong at the top of the `main.rs` for the executable. Where do you think they should go?

Comment (LucasSte) [reply src/bin/solang.rs:882]: Perhaps they belong in another file.

Comment (seanyoung) [reply src/bin/solang.rs]: You are right. It was even worse: the structures were defined in different places. I've created `src/standard_json.rs`.

File: src/standard_json.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+//! This module defines the json format for `solang compile --standard-json`.
```
Comment (LucasSte) [inline src/standard_json.rs:1]: Please add the SPDX header: `// SPDX-License-Identifier: Apache-2.0`

**Review by LucasSte — CHANGES_REQUESTED** [review]

**Review by xermicus — APPROVED** [review]

File: src/standard_json.rs (hunk 2 above)
Comment (seanyoung) [reply src/standard_json.rs:3]: Good point, fixed!

File: src/bin/solang.rs [hunk 3]
```diff
@@ -878,19 +858,22 @@ fn imports_arg(matches: &ArgMatches) -> FileResolver {
...
+                eprintln!("error: import path '{}': {}", path, e);
                 std::process::exit(1);
             }
         }
     }
 
     resolver
 }
+
+fn parse_import_map(map: &str) -> Result<(String, String), String> {
```
Comment (LucasSte) [inline src/bin/solang.rs:873]: This function deserves a comment. I see you use it in `ValueParser::new(parse_import_map)`, but It is not clear what these three returned strings represent. If someone were to re-use your function, they would need to know what it does.

**Review by LucasSte — CHANGES_REQUESTED** [review]
Sorry, I missed this last time.

File: src/bin/solang.rs (hunk 3 above)
Comment (seanyoung) [reply src/bin/solang.rs]: Good point. I've added a comment

**Review by LucasSte — APPROVED** [review]
